### ISSUE TITLE
MetaMask error "Attempting to use a disconnected port object" causes dApp hang

### ISSUE THREAD

**Issue by nuurcodes** [MetaMask/metamask-extension#11730]
**Describe the bug**
If you connect your wallet to a [dApp](https://app.mistx.io/#/exchange) on Chrome and then close the browser, you will get the following error when re-opening Chrome:

```
:3000/:1 Unchecked runtime.lastError: Could not establish connection. Receiving end does not exist.

Uncaught Error: Attempting to use a disconnected port object
    at onloadwff.js:71
    at onloadwff.js:71
```

**Steps to reproduce (REQUIRED)**
Steps to reproduce the behavior, libraries used with version number, and/or any setup information to easily reproduce:

1. Change Chrome "On startup" settings to "Continue where you left off"
2. Open a dApp, like https://app.mistx.io/#/exchange
3. Quit Chrome
4. Open Chrome
5. dApp is now hanging, console shows the above error

The app works when you refresh the page

**Browser details (please complete the following information):**
 - OS: OS x
 - Browser: [Chrome Version  92.0.4515.107 (Official Build) (x86_64)]
 - MetaMask Version: [9.8.4]

**Comment by joehawary** [comment]
You can replicate this by closing the browser and opening a new session so MM is asking for the unlock password. After entering the password the issue remains the same. Tracing the state of MM, it doesn’t change and is still giving isAuthorizated even though the wallet is not active

**Comment by debrice** [comment]
@nuurcodes and @joehquak , the onloadwff.js comes from the lastpass extension.

**Comment by vandan** [comment]
Should be closed with #14987
